File: cmptext.py
# ファイル名を指定
afile = "./fizzbuzz_python.txt"
bfile = "./fizzbuzz_rust.txt"

# ファイルを文字列として読み込む
with open(afile, "r") as fp:
    astr = fp.read()
with open(bfile, "r") as fp:
    bstr = fp.read()

# 念のため余分な空行をトリム
astr = astr.strip()
bstr = bstr.strip()

# 比較 ---
if astr == bstr:
    print("OK")
else:
    print("NG")

File: cmptext.rs
// ファイル操作のライブラリーを読む
use std::fs;
fn main() {
  // ファイル名を指定
  let afile = "./fizzbuzz_python.txt";
  let bfile = "./fizzbuzz_rust.txt";

  // ファイルを文字列として読み込む
  let astr = fs::read_to_string(afile).unwrap();
  let bstr = fs::read_to_string(bfile).unwrap();

  // 年のためトリム
  let astr = astr.trim();
  let bstr = bstr.trim();

  // 比較
  if astr == bstr {
    println!("OK");
  } else {
    println!("NG");
  }
}

